Source/Scene/Model.js

```diff
@@ -119,7 +119,7 @@ define([
 
         this.decompressedViewsToCreate = new Queue();
         this.decompressedViews = {};
-        this.decompressionInFlight = 0;
+        this.decompressionInFlight = false;
```

Comment:
> That all being said, turning this back into a counter and using multiple workers is really easy, so if you want me to do it (for performance experiments), it's probably only about 5 minutes worth of work.

Yes, this would be useful for our testing (think loading a model with everything else turned off) even if it is set to one worker in practice.